File: src/test/scala/test/Java/GetLocalHost.java
package test.Java;

import java.net.InetAddress;
import java.net.UnknownHostException;

public class GetLocalHost {

    public static void main (String[] ags) throws UnknownHostException {
        InetAddress address = InetAddress.getLocalHost();
        System.out.println(address.getHostAddress());
        System.out.println(address.getAddress());
        System.out.println(address.getCanonicalHostName());
        System.out.println(address.getHostName());
    }

}

File: src/test/scala/test/scala/GetLocalHost.scala
package test.scala

import java.net.InetAddress

object GetLocalHost {

  def main(args: Array[String]): Unit = {
    val address = InetAddress.getLocalHost
    println(address.getAddress)
    println(address.getHostAddress)
    System.out.println(address.getHostAddress());
    System.out.println(address.getAddress());
    System.out.println(address.getCanonicalHostName());
    System.out.println(address.getHostName());
  }

}
